proof/region/v1/tests/test_workflow_admission.py

```
@@ -0,0 +1,169 @@
+        for name in (
+            "replay the bounded full-domain prefix through the shard runner",
+            "replay one full-domain window lane",
+        ):
+            script = step_script("full-domain-corpus.yml", name)
+            self.assertNotRegex(script, r"\$\{\{[^}]*inputs\.")
+            with tempfile.TemporaryDirectory() as directory:
+                root = Path(directory)
+                stub = root / "python3"
+                stub.write_text('#!/bin/sh\nprintf "%s\\0" "$@" > "$CAPTURE"\n')
+                stub.chmod(0o700)
+                coordinates = {key: "'; printf injected > marker; #" for key in ("POINTS", "SHARD_POINTS", "WINDOW_START", "WINDOW_POINTS")}
```

Ruff (0.16.3)

[warning] Unnecessary dict comprehension for iterable; use `dict.fromkeys` instead

Replace with `dict.fromkeys(iterable)`)

(C420)